src/Utilities/Compiler/Options/MSBuildPropertyOptionNames.cs

```diff
@@ -15,6 +15,8 @@ namespace Analyzer.Utilities
     internal static class MSBuildPropertyOptionNames
     {
         public const string TargetFramework = nameof(TargetFramework);
+        public const string TargetFrameworkIdentifier = nameof(TargetFrameworkIdentifier);
+        public const string TargetFrameworkVersion = nameof(TargetFrameworkVersion);
```

Comment:
This is what causes the `CompilerVisibleProperty` items to be generated in the props/targets that are inserted into the SDK and is what causes the `build_property.*` metadata to be automatically visible to analyzers by default.